typings/react-live.d.ts

```diff
@@ -18,6 +18,12 @@ export type LiveProviderProps = Omit<DivProps, 'scope'> & {
   code?: string;
   noInline?: boolean;
   transformCode?: (code: string) => string;
+  language?: string;
+  disabled?: boolean;
+  theme?: {
```

Comment:
It'd probably make sense to modify `prism-react-renderer`'s typings so that we can just use its `PrismTheme` type, especially since it doesn't support all of `CSSProperties`

Reply:
Yeah, it's true. I gotta admit, I didn't really put much thought into it yesterday.